docs/1.-Welcome/Documentation-Guidelines/Other Documentation Types/Readme-files-in-code-repositories.md

```diff
@@ -0,0 +1,83 @@
+---
+title: README files in code repositories
+authors: 
+- Dave Arthur 
+reviewed: 
+reviewer:
+next-review: 01-05-2022
+---
+
+# README files
+
+Every repository should have a file called `README.md` in the root folder. Using [markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) files instead of a plain text file allows us to format documentation, add links and images to enrich the documentation. As an added bonus, they get rendered nicely when browsing the repository via the GitHub web UI.
+
+## Required sections
+
+!!! note
+
+    The layout for our README files is based loosely on the [Best README Template](https://github.com/othneildrew/Best-README-Template) project.
+
+    The Azure DevOps [README guidelines](https://docs.microsoft.com/en-us/azure/devops/repos/git/create-a-readme?view=azure-devops) are also a useful reference.
+
+Every README file should have the following sections:
+
+- Overview
+- Built with
+- Getting started
+- Usage
+- Contributing
+
+Whilst these sections are a minimum, and should appear on the README for every repository we manage, they don't need to be lengthy! A brief summary or link out to documentation that exists elsewhere is acceptable.
+
+### Overview
+
+Provide a brief overview of the repository, and the problem it aims to solve.
+
+### Built with
+
+List the major technologies and frameworks used within the repository, as a bullet-point list. Ideally link out to some documentation for each item, this is especially important for technologies and frameworks outside our usual tech stack.
+
+### Getting started
+
+This section would describe how someone who has just downloaded the repository should get started running/using it. Include any pre-requisites or frameworks that must be installed locally, and any additional installation steps where relevant.
```

Comment:
This bit I would probably push a bit harder on. This section of a readme is often the one that has the greatest impact in terms of picking up the repo, and getting it running ready for dev work.

Perhaps:
> 
> This section **must clearly** describe how someone who has just downloaded the repository can get started running/using it. **It  must** include any pre-requisites or frameworks that must be installed locally, and any additional installation steps where relevant. 
> 
> *- Never make any assumptions about the engineer's local environment or tools.
> - Make sure the instructions are step-by-step*

Reply:
Nice wording - have adapted this.